packages/hash/frontend/src/shared/layout/layout-with-sidebar/account-page-list/page-tree-item.tsx

```diff
@@ -91,40 +94,32 @@ const CustomContent = forwardRef((props: TreeItemContentProps, ref) => {
       >
         <FontAwesomeIcon icon={faChevronRight} />
       </IconButton>
-
-      <PageIcon
+      <PageIconButton
         hasDarkBg={selected}
         accountId={accountId}
         entityId={nodeId}
         size="small"
       />
-
-      <Link
-        noLinkStyle
-        tabIndex={-1}
-        sx={{
-          flex: 1,
+      <Typography
+        variant="smallTextLabels"
+        className="page-title"
+        sx={({ palette }) => ({
           ml: "6px",
-        }}
-        href={url}
+          display: "block",
+          color: palette.gray[70],
+          fontWeight: 400,
+          py: 1,
+          overflow: "hidden",
+          textOverflow: "ellipsis",
+          whiteSpace: "nowrap",
```

Comment:
I think a good addition will be to have a tooltip that shows the full page title when the text is hovered.

Reply:
https://app.asana.com/0/1201095311341924/1202861224066305/f